lib/situation_room/check/manager.ex

```diff
@@ -0,0 +1,14 @@
+defmodule SituationRoom.Check.Manager do
+  @moduledoc """
+  Logic for sending requests to site URLs
+  """
+  alias SituationRoom.Check.GetRequest
+  alias SituationRoom.Check.Outage
+  alias SituationRoom.Checks
+
+  def all_checks(site) do
+    GetRequest.run(site.endpoint, site.id)
```

Comment:
If we're gonna let the thing that does requests know about the site id as well, maybe we just pass `site` itself here instead of breaking it up.